### PR TITLE
fix(curriculum): updated seed text in cat photo app

### LINKED ISSUE #47854: Step 12 Minor Tweek
### Describe the Issue

The changes to Step 12 on Cat Photo App seem to be paying off. 

One last common issue seems to be the period inside of the anchor text. Should we add a dynamic hint? 

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-12

### Your code

```html
<html>
  <body>
    <h1>CatPhotoApp</h1>
    <main>
      <h2>Cat Photos</h2>
      <!-- TODO: Add link to cat photos --> 
      <p>Click here to view more <a href="https://freecatphotoapp.com"> cat photos.</a></p> 
      <a href="https://freecatphotoapp.com">link to cat pictures</a>
      <img src="https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg" alt="A cute orange cat lying on its back.">
    </main>
  </body>
</html>
```

### Expected behavior

Clear hint indicating to remove the period.

### System

All

### PR DESCRIPTION
**Pull request by nayabatir1**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47854

### CODE REVIEW

**Review by Sboonny — COMMENTED** [review]
Unfortunately, you need to change the files with the seed code of `<p>Click here to view more <a target="_blank" href="https://freecatphotoapp.com">cat photos</a>.</p>` and a like, as well.

**Comment by Sboonny** [comment]
https://github.com/nayabatir1/freeCodeCamp/blob/f8fcbda51ec175e0e8bbfef4a34b9c491403e0db/curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md#L35-L42?plain=1

This should sort it, and hopefully pleases the curriculum test. Didn't test it yet tho

> <p>See more <a target="_blank" href="https://freecatphotoapp.com">cat photos</a> in our gallery.</p>

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc17dc8f86c76b9248c6eb5.md [hunk 1]
```diff
@@ -62,9 +62,7 @@ assert(
     <h1>CatPhotoApp</h1>
     <h2>Cat Photos</h2>
 --fcc-editable-region--
-    
-    <p>Click here to view more cat photos.</p>
-    
+    <p>See more cat photos in our gallery.</p>
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc17dc8f86c76b9248c6eb5.md:65]:
```suggestion

    <p>See more cat photos in our gallery.</p>
    
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc2385ff86c76b9248c6eb7.md [hunk 2]
```diff
@@ -64,8 +64,7 @@ assert(
     <h1>CatPhotoApp</h1>
     <h2>Cat Photos</h2>
     <!-- TODO: Add link to cat photos -->
-    <p>Click here to view more cat photos.</p>
-  
+    <p>See more cat photos in our gallery.</p>
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc2385ff86c76b9248c6eb7.md:67]:
```suggestion
    <p>See more cat photos in our gallery.</p>
    
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23f9bf86c76b9248c6eba.md [hunk 3]
```diff
@@ -52,8 +52,7 @@ assert(collection.indexOf('P') < collection.indexOf('IMG'));
       <h2>Cat Photos</h2>
       <!-- TODO: Add link to cat photos -->
 --fcc-editable-region--
-      <p>Click here to view more cat photos.</p>
-      
+      <p>See more cat photos in our gallery.</p>
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23f9bf86c76b9248c6eba.md:55]:
```suggestion
      <p>See more cat photos in our gallery.</p>
      
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24614f86c76b9248c6ebd.md [hunk 4]
```diff
@@ -69,8 +69,7 @@ assert(
       <h2>Cat Photos</h2>
       <!-- TODO: Add link to cat photos -->
 --fcc-editable-region--
-      <p>Click here to view more cat photos.</p>
-      
+      <p>See more cat photos in our gallery.</p>
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24614f86c76b9248c6ebd.md:72]:
```suggestion
      <p>See more cat photos in our gallery.</p>
      
```

**Review by Sboonny — CHANGES_REQUESTED** [review]
there are two seed code left, one in step 14 https://github.com/nayabatir1/freeCodeCamp/blob/f8fcbda51ec175e0e8bbfef4a34b9c491403e0db/curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa2407b521be39a3de7be1.md#L48?plain=1

The second which making the test fails is in step 13 https://github.com/nayabatir1/freeCodeCamp/blob/f8fcbda51ec175e0e8bbfef4a34b9c491403e0db/curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md#L77

----

About the deleted spacing, they were added as a support for beginners, deleting them is adding an issue to the code seed again

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc17dc8f86c76b9248c6eb5.md [hunk 5]
```diff
@@ -62,8 +62,8 @@ assert(
     <h1>CatPhotoApp</h1>
     <h2>Cat Photos</h2>
 --fcc-editable-region--
-    
-    <p>Click here to view more cat photos.</p>
+
```
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc17dc8f86c76b9248c6eb5.md:65]:
```suggestion
    
```

**Comment by raisedadead** [comment]
We are officially on Node.js 18 now. We have changed the merge requirements accordingly.

You will need to rebase this PR against the main branch OR Click the "Update branch" button to get the new checks going.

Thanks for your contribution.

**Review by gikf — CHANGES_REQUESTED** [review]
Test in step 4 needs to be changed to conform with new text:

https://github.com/freeCodeCamp/freeCodeCamp/blob/04fcc71bc943aa7b0cf77e06c9819ea98fdbea2a/curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc17dc8f86c76b9248c6eb5.md?plain=1#L50

Should be

```js
/<!--todo:addlinktocatphotos--><p>seemorecatphotosinourgallery\.?<\/p>/i
```

**Review by gikf — COMMENTED** [review]
This should be the last failing test:

https://github.com/freeCodeCamp/freeCodeCamp/blob/889eb40fcc78016a9378a65eadf0ceefb78e2b86/curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23991f86c76b9248c6eb8.md?plain=1#L75

```
    .match(/see\s+more\s+cat\s+photos\s+in\s+our\s+gallery\.?$/)
```

There's also step 15, having still the old text https://github.com/freeCodeCamp/freeCodeCamp/blob/889eb40fcc78016a9378a65eadf0ceefb78e2b86/curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa30b9eacea3f48c6300ad.md?plain=1#L80

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Review by Sboonny — APPROVED** [review]

**Comment by Sboonny** [comment]
Thank you for the hard work. 🥳

**Pull request merged**